uPortal-soffit-renderer/src/main/java/org/apereo/portal/soffit/renderer/SoffitRendererController.java

```diff
@@ -0,0 +1,294 @@
+package org.apereo.portal.soffit.renderer;
```

Comment:
I've just run `$mvn license:format` on the whole project.  It picked up that, plus a hand full of other items.  Soon to be committed.